engine/src/main/java/org/terasology/engine/EngineStatus.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.engine;
+
+/**
+ * Engine Status provides the current status of the engine - what it is doing, at a higher granularity than just running. This can be used by external and internal observers
+ * to report on the state of the engine, such as splash screens/loading screens.
+ *
+ * @author Immortius
+ */
+public interface EngineStatus {
+
+    /**
+     * @return The description of the status
+     */
+    String getDefaultDescription();
```

Comment:
Status changes are intended to potentially be displayed to users - both in the splash screen (already implemented) and other load screens.

I probably will just make it english only, and remove default from the name. Translation can either use the common technique of using the english string to map to a language appropriate string, or can use the status object to map to a language appropriate string. Or the strings can come from language property files eventually. Either way there won't be any other methods so no need to have default in the name.